index.js

```diff
@@ -3,6 +3,11 @@ const github = require('@actions/github')
 const octokitCore = require('@octokit/core')
 const paginate = require('@octokit/plugin-paginate-rest')
 
+
+function hasValue(string) {
+    return string != null && string.trim().length != 0
```

Comment:
use strict equality to not accidentally handle truthies/falsies
`return string !== null && string.trim().length !== 0`

Reply:
OKAY